src/Domain/Setting/SimpleSetting.ts

```diff
@@ -0,0 +1,9 @@
+import { Uuid } from '../Uuid/Uuid'
+import { Setting } from './Setting'
+
+export type SimpleSetting = Omit<Setting, 
+  'user' | 
+  'createdAt' | 
+  'updatedAt' | 
+  'serverEncryptionVersion'
+> & { userUuid: Uuid }
```

Comment:
@karolsojko I should have not included the `& { userUuid: Uuid }` -- it is indeed not necessary, as you mentioned. Will remove in the next PR.